assets/launcher/lang/sklmessages_pl_PL.properties

```
@@ -101,16 +101,16 @@ settings.name=Ustawienia launchera
 settings.button.save=Zapisz
 settings.button.cancel=Anuluj
 settings.label.title=Ustawienia launchera
-settings.link.translate=Pomóż tłumaczyć
+settings.link.translate=Pomóż tłumaczyć SKlaunchera
```

"Pomóż tłumaczyć SKlaunchera" brzmi po prostu 🤢

Lepiej by brzmiało
```suggestion
settings.link.translate=Pomóż tłumaczyć SKlauncher
```
lub
```suggestion
settings.link.translate=Pomóż tłumaczyć launcher
```

---

Pierwsza opcja najlepsza

---

I ją zastosowałem